### ISSUE TITLE
[BUG] URL Not Centered

### ISSUE THREAD

**Issue by amcottre** [AdamXweb/WhiteSurFirefoxThemeMacOS#121]
### Describe the problem

I didn't see anything about this but it looks like the URL is off center for me, either lining up w/the left or right edge of the notch depending on how I have the toolbar setup. I tried adding the flexible spaces in the tool bar but it just pushes it further to one side or the other. Even removing all the flexible space its still off (I attached the picture comparing it to safari and you can see how it is compared to the center line between the tabs). Is there a way to better center it? 

### Version

1.5.5

### OS

MacOS 14.5

### Firefox Version

128.0.2

### Install Options

none

### Colour Theme

none

### Additional info

<img width="1766" alt="Screenshot 2024-07-25 at 9 54 27 PM" src="https://github.com/user-attachments/assets/05b4340a-d962-49c0-a287-c4ca4f452579">

### Please tick the boxes

- [X] You are using a [current](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/releases/) version of the theme. (check the date associated with the version)
- [X] You've checked that this [issue hasn't already been raised](https://github.com/adamxweb/WhiteSurFirefoxThemeMacOS/issues?q=is%3Aissue)

**Comment by AdamXweb** [comment]
Hi @amcottre, thanks for raising this bug.
Something to note with the alignment are two items:
1. The URL tries to center based on the container it is given, which means between the padlock on the left, and the star on the right. I may look into removing the tracking protection icon container with the shield as an install option to make it cleaner. It could differ based on your extensions, but i see the appeal on it being centered all the time.
FYI, this is what I see when debugging the browser to check alignments. <img width="926" alt="Screenshot 2024-07-26 at 12 56 25 pm" src="https://github.com/user-attachments/assets/5dc28fb5-bdcf-452c-8ea3-9632e626979e">

2. It's a bit of a visual difference, as Safari omits the protocol i.e. https when looking at the URL. There is a hidden config item under `browser.urlbar.trimHttps` in `about:config` that you can change to true to display like the below. (i'm not a fan as it doesn't show the entire URL)
<img width="848" alt="Screenshot 2024-07-26 at 1 00 35 pm" src="https://github.com/user-attachments/assets/1931b11f-10a3-4855-b334-467bfbb210c7">

I'll rename this as a feature to add the option to better center it.
Please let me know if this doesn't fix the centering on your device

**Comment by amcottre** [comment]
Appreciate the super quick reply. It look like it might be the positioning of the entire URL bar and not the URL itself. I did the 2nd option which I do prefer, but it's still shifted right. I think the URL bar is being centered between the right end of the close buttons space and the far right side instead of the far left/right sides. It seems like adding the sidebar button to the left is what mostly causes it. I took a rough "measurement" of the right side and put it over the left below

![Screenshot 2024-07-26 at 9 15 09 AM](https://github.com/user-attachments/assets/b48e799f-23b8-49f0-893d-a5efda081b3a)
